recipe/meta.yaml

```diff
@@ -23,14 +23,16 @@ build:
 
 requirements:
   host:
-    - python >=3.9
+    - python {{ python_min }}
```

Comment:
Should be equivalent anyway, as defined in:
https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/1ae12810d4b2f2104a3862a1d11113fb8b4981a7/recipe/conda_build_config.yaml#L851-L854